modules/gapi/include/opencv2/gapi/infer/onnx.hpp

```diff
@@ -58,6 +58,8 @@ struct ParamDesc {
     PostProc custom_post_proc;
 
     std::vector<bool> normalize;
+
+    std::vector<std::string> d_out_names;
```

Comment:
It make sense to add comment with decoding of field name

Reply:
Added comment.